src/dut.rs

```diff
@@ -123,12 +169,20 @@ impl MonitoredDut {
             match child.try_status()? {
                 None => {
                     self.reconnecting = false;
-                    Ok(format!(
+                    let mut status = format!(
```

Comment:
the below would also get read of the need for mutation

Reply:
Got it.